benchmark/Streamly/Benchmark/Data/Stream/StreamK.hs

```diff
@@ -471,7 +480,7 @@ sourceConcatMapId val n =
 {-# INLINE concatMapBySerial #-}
 concatMapBySerial :: Int -> Int -> Int -> IO ()
 concatMapBySerial outer inner n =
-    S.drain $ S.concatMapBy S.serial
+    S.drain $ S.concatMapWith S.serial
```

Comment:
Are `concatMapWith` and `concatMapBy` same?

Reply:
concatMapBy is older name, later renamed to concatMapWith.